From 6d9e51316c76e07f43f84e5f0e1b6de26180ae1f Mon Sep 17 00:00:00 2001
From: Roman Shaposhnik <rvs@cloudera.com>
Date: Fri, 16 Dec 2011 15:07:15 -0800
Subject: [PATCH 024/795] MR1: CLOUDERA-BUILD. Disabling missing artifacts in yet another profile
 (cherry picked from commit d0c2cf59e74144e0f0cd81b85410b442e4ad11d7)

(cherry picked from commit f7af8d725723789c9d8157a1b539a44fca25016b)
---
 cloudera/maven-packaging/pom.xml |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/cloudera/maven-packaging/pom.xml b/cloudera/maven-packaging/pom.xml
index 1772c2c..b032243 100644
--- a/cloudera/maven-packaging/pom.xml
+++ b/cloudera/maven-packaging/pom.xml
@@ -102,11 +102,11 @@
       <modules>
         <module>../../cloudera-pom.xml</module>
         <module>hadoop-core</module>
-        <module>hadoop-streaming</module>
-        <module>hadoop-mrunit</module>
+        <!-- module>hadoop-streaming</module>
+        <module>hadoop-mrunit</module -->
         <module>hadoop-examples</module>
         <module>hadoop-test</module>
-        <module>hadoop-tools</module>
+        <!-- module>hadoop-tools</module -->
         <module>hadoop-tarball</module>
       </modules>
     </profile>
-- 
1.7.0.4

